### ISSUE TITLE
Usage of synced templates is tracked in ET

### ISSUE THREAD

**Issue by MaxTru** [camunda/camunda-modeler#1968]
> **This is an epic issue which may be broken down into multiple tasks, if needed**

__What should we do?__

In order to understand the users need, the E.T. tracking currently includes already [tracking of the element template usage](https://github.com/camunda/camunda-modeler/tree/develop/docs/telemetry). This element template tracking is to be improved in order to understand: 

- [ ] how many element templates are present 
- [ ] how many element template are applied 
- [ ] how many element templates were synced from a Cawemo Catalog project. 

- [ ] Before starting the implementation, make sure to align with E.T. team, process is decribed here: https://confluence.camunda.com/display/camBPM/ET+-+Easy+Telemetry

(*note that these questions might change based on kickoff / clarification of this issue*)

__Why should we do it?__

We would like to understand whether the added features from 7.14. are useful for the user and adopted.

**Comment by MaxTru** [comment]
Moving to backlog since this is not a priority for Q1
